src/sensai/data_transformation/dft.py

```diff
@@ -510,28 +512,32 @@ def __init__(self,
+                    * the resulting rule will match only a single column. Otherwise, ``independent_columns``
+                      must be specified to True or False.
 
             :param skip: flag indicating whether no transformation shall be performed on all of the columns (because they are already
                 normalised)
             :param unsupported: flag indicating whether normalisation of all columns is unsupported (shall trigger an exception if
                 attempted)
-            :param transformer: a transformer instance (from sklearn.preprocessing, e.g. StandardScaler) to apply to the matching column(s)
+            :param transformer: a transformer instance (following the sklearn.preprocessing interface, e.g. StandardScaler) to apply to the matching column(s)
                 for the case where a transformation is necessary (skip=False, unsupported=False). If None is given, either
-                transformerFactory or the containing instance's default factory will be used.
-                NOTE: Use an instance only if you want, in particular, the instance to be shared across several models that use the same
-                feature with associated rule/rule template (disabling `fit` where appropriate). Otherwise, use a factory.
+                transformer_factory or the containing ``DFTNormalisation`` instance's default factory will be used when the normaliser is
+                fitted.
+                NOTE: Using a transformer_factory is usually preferred. Use an instance only if you want the
+                same transformer instance to be used in multiple places - e.g., sharing it across several models that use the same
+                column with associated rule/rule template (disabling `fit` where appropriate).
             :param transformer_factory: a factory for the generation of the transformer instance, which will only be applied if
-                `transformer` is not given; if neither `transformer` nor `transformer_factory` are given, the containing instance's default
+                `transformer` is not given; if neither `transformer` nor `transformer_factory` are given, the containing ``DFTNormalisation`` instance's default
                 factory will be used. See :class:`SkLearnTransformerFactoryFactory` for convenient construction options.
-            :param independent_columns: whether, for the case where the rule matches multiple columns, the columns are independent and a
-                separate transformation is to be learned for each of them (rather than using the same transformation for all columns and
-                learning the transformation from the data of all columns); must be specified for rules matching more than one column,
-                None is acceptable only for a single column
+            :param independent_columns: only relevant if the resulting rule matches multiple columns, in which case it is required.
```

Comment:
The old docstring was better - more to the point. The docstring should always mention the main semantics first. Preconditions for applicability should come afterwards or - as in the original docstrings - can be interjected.
I'm not sure we need to mention that independence does not mean a separate transformer instance, as this is a technicality; all that matters is that the transformation will be learnt independently.

Reply:
Let's chat at the office at some point how we can adjust this. I sat for about 10 minutes trying to understand what this means, and finally had to write some quick test and also look into the source code to finally understand what exactly this did

Reply:
Done, added a block-comment in _fit